transport-native-epoll/src/main/java/io/netty/channel/unix/Socket.java

```diff
@@ -428,4 +437,5 @@ private static native DatagramSocketAddress recvFromAddress(
     private static native void setSoLinger(int fd, int soLinger) throws IOException;
     private static native void setTcpDeferAccept(int fd, int deferAccept) throws IOException;
     private static native void setTcpQuickAck(int fd, int quickAck) throws IOException;
+    private static native PeerCredentials peerCredentials(int fd) throws IOException;
```

Comment:
https://github.com/netty/netty/pull/5940#discussion_r86203348

Sorry for the conflicting direction ... but please add the `get` prefix for consistency.